mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -338,6 +338,42 @@ std::string getTuningProblemStr(ModuleOp &mod) {
   } else if (opType == KernelType::Gemm) { // gemm case
     rock::GemmOp rGemmOp = dyn_cast<rock::GemmOp>(gemmOp);
     // Please keep these in sync with mlir/utils/performance/perfRunner.py
+    // Data type
+    problemOS << "-t ";
+    Type elemTypeA = gemmIF.getAType(), elemTypeB = gemmIF.getBType();
+    if (elemTypeA.isF32() && elemTypeB.isF32()) {
```

Comment:
same suggestion as above: can we only add i8i8 and i8i32?

Reply:
We don't handle i8i8 or i8i32 here. This is the rocmlir-gen argument for filter and input data types.